src/test/librbd/journal/test_Replay.cc

```diff
@@ -103,6 +103,16 @@ class TestJournalReplay : public TestFixture {
     ictx->journal->open(&open_cond);
     ASSERT_EQ(0, open_cond.wait());
   }
+
+  bool is_rbd_rwl_enabled() {
```

Comment:
Nit: second time I've seen this function -- can we move it to `test_support`?

Reply:
DONE